python/setup.py

```diff
@@ -401,6 +444,11 @@ def get_proton_cmake_args(self):
     def build_extension(self, ext):
         lit_dir = shutil.which('lit')
         ninja_dir = shutil.which('ninja')
+        if platform.system() == "Windows":
+            vs_path = find_visual_studio(["[17.0,18.0)", "[16.0,17.0)"])
+            env = set_env_vars(vs_path)
+            if not vs_path:
+                raise EnvironmentError("Visual Studio 2019 or 2022 not found.")
```

Comment:
It's probably a good idea to define `initialize_visual_studio_env` as in `CLFinder.py` file here as well.
```suggestion
            if not vs_path:
                raise EnvironmentError("Visual Studio 2019 or 2022 not found.")
            env = set_env_vars(vs_path)
```